{{ cookiecutter.project_name }}/{{ cookiecutter.app_name }}/urls.py

```diff
@@ -8,6 +8,6 @@
     url(
         r'^$',
         TemplateView.as_view(template_name='{{ cookiecutter.app_name }}/index.html'),
-        name='{{ cookiecutter.app_name }}_index'
+        name='{{ cookiecutter.app_name }}-index'
```

Comment:
This change is just a style change, right? It doesn't actually fix anything with regards to the cookiecutter-generated stuff. Just checking to make sure I understand.